bin/percy

```diff
@@ -10,24 +10,24 @@ def system!(*args)
 end
 
 FileUtils.chdir APP_ROOT do
+  puts "Enter a 4 digit PR number if available (or just hit enter)"
+  pr_number = gets.chomp
+
   system!({ "PERCY_TOKEN" => Rails.application.credentials.percy_token },
     '
       if [ -z $PERCY_TOKEN ]; then echo "== 🚨 Please download development.key to access PERCY_TOKEN =="; exit 1; fi
 
-      if [ -z "$(git status --porcelain)" ]; then
-        echo "== 📸 Take baseline images =="
-        git checkout main
-        bundle install
-        npx percy exec -- rspec --tag screenshot
-        echo "== 📸 Take new images =="
-        git checkout -
-        bundle install
-        npx percy exec -- rspec --tag screenshot
-      else
-       echo "== 🚨 Please commit changes first =="
-       exit 1
-      fi
-    '
+      if [ -n "$(git status --porcelain)" ]; then echo "== 🚨 Please commit changes first =="; exit 1; fi
+
+      echo "== 📸 Take baseline images =="
+      git checkout main
+      bundle install
+      npx percy exec -- rspec --tag screenshot
+      echo "== 📸 Take new images =="
+      git checkout -
+      bundle install
+      PERCY_TARGET_BRANCH=main PERCY_PULL_REQUEST=%{pr_number} npx percy exec -- rspec --tag screenshot
```

Comment:
I apply these env variables only on the new branch execution. Since [these are optional](https://docs.percy.io/docs/environment-variables#optional), `PERCY_PULL_REQUEST` can be empty when running the script without giving it a PR number.